README.md

```diff
@@ -292,28 +292,7 @@ Amongst the many awesome contributors, there are the maintainers. These maintain
 * Committing to reviewing pull requests, issues, and addressing comments/questions as quickly as possible
 * Acting as a point of contact for questions
 
-Just tag **@intelsdi-x/snap-maintainers** if you need to get some attention on an issue. If at any time, you don't get a quick enough response, reach out to any of the following team members directly:
```

Comment:
@mjbrender, could You explain me why the table with snap-maintainers has been removed?